core/src/main/java/org/apache/iceberg/AllManifestsTable.java

```diff
@@ -292,11 +295,14 @@ static StaticDataTask.Row manifestFileToRow(
         manifest.partitionSpecId(),
         manifest.snapshotId(),
         manifest.content() == ManifestContent.DATA ? manifest.addedFilesCount() : 0,
+        manifest.content() == ManifestContent.DATA ? manifest.addedRowsCount() : 0L,
         manifest.content() == ManifestContent.DATA ? manifest.existingFilesCount() : 0,
+        manifest.content() == ManifestContent.DATA ? manifest.existingRowsCount() : 0L,
         manifest.content() == ManifestContent.DATA ? manifest.deletedFilesCount() : 0,
         manifest.content() == ManifestContent.DELETES ? manifest.addedFilesCount() : 0,
         manifest.content() == ManifestContent.DELETES ? manifest.existingFilesCount() : 0,
         manifest.content() == ManifestContent.DELETES ? manifest.deletedFilesCount() : 0,
+        manifest.content() == ManifestContent.DELETES ? manifest.deletedRowsCount() : 0L,
```

Comment:
Is this correct? I don't think this captures copy-on-write deletes (that I would expect to be included when seeing a column called `deleted_rows_count`).